ios/kmbuild.sh

```diff
@@ -37,6 +37,7 @@ display_usage ( ) {
     echo "  -upload-sentry          Uploads debug symbols, etc, to Sentry"
     echo "  -debug                  Sets the configuration to debug mode instead of release."
     echo "  -download-resources     Download up-to-date versions of the engine's default resources from downloads.keyman.com."
+    echo "  -all-artifacts          Produces all build artifacts, including a Simulator-installable copy of the .app"
```

Comment:
Could we call this `-simulator-target`? Given that's more explicit about what it actually does:

```suggestion
    echo "  -simulator-target       Also builds a Simulator-installable copy of the .app"
```

Reply:
I was back and forth a while on the nomenclature.  My worry there:  we want to build _all_ targets, not _just_ the simulator target.  And I could see that being a very logical inference someone unfamiliar with the scripts would make.

Reply:
I thought about `-also-simulator-target` but that was too much, and felt that `-all-artifacts` was just a bit vague... The description explains it anyway so it's not critical but I prefer flags to be narrow in meaning if possible.

Reply:
How does `-add-sim-artifact` sound?  Trying to keep it reasonably short, hence the shortened 'sim' over 'simulator'.